src/operun/crm/config.py

```diff
@@ -5,9 +5,9 @@
 
 
 ACCOUNT_TYPES = SimpleVocabulary(
-    [SimpleTerm(value=u'customer', title=_(u'Customer')),
-     SimpleTerm(value=u'vendor', title=_(u'Vendor')),
-     SimpleTerm(value=u'partner', title=_(u'Partner')),
-     SimpleTerm(value=u'prospect', title=_(u'Prospect')),
-     SimpleTerm(value=u'contact', title=_(u'Contact'))]
-    )
+    [SimpleTerm(value=u'contact', title=_(u'Contact')),
+     SimpleTerm(value=u'employee', title=_(u'Employee')),
+     SimpleTerm(value=u'lead', title=_(u'Lead')),
+     SimpleTerm(value=u'customer', title=_(u'Customer')),
+     SimpleTerm(value=u'vendor', title=_(u'Vendor'))]
```

Comment:
Prospect was available on the old version of this. We need to handle it. Keep prospect as value or write migration for this.